design/ceph-mixins.md

```diff
@@ -0,0 +1,61 @@
+# Monitoring ceph storage with ceph-mixins
+
+---
+
+## Overview
+
+Currently, rook deploys ceph storage but not ceph monitoring. It can be deployed manually using documented steps. But still, it lacks Prometheus Alerts and Recording Rules that is useful in easy monitoring of ceph storage.
+
+This is where **ceph-mixins** comes in. Ceph-mixins defines a minimalistic approach to package together the ceph storage specific prometheus alerts, recording rules and grafana dashboards, in a simple and platform agnostic way.
+
+Rook can use ceph-mixins as the default solution for monitoring ceph storage.
+
+## Design
```

Comment:
One overall question is whether we can support different rules for different releases of Ceph? For example, would there be any alerts different between Mimic and Nautilus? If so, could we generate two different versions of the yaml?

Reply:
@travisn It is possible and we do plan to appropriately tag each release.